cloudhub-2/modules/ROOT/pages/ps-gather-setup-info.adoc

```diff
@@ -373,7 +373,10 @@ The CIDR block must be unique across all private network connections.
 
 MuleSoft recommends using subnets from standard RFC 1918 addresses, such as `10.0.0.0/8`, `172.16.0.0/12`, and `192.168.0.0/16`.
 
-You cannot use the following reserved CIDR blocks:
+[[reserved-cidr]]
+==== Reserved and Unusable CIDR
+
+You cannot use the following list of CIDR blocks for reserved CIDR blocks or as private space network CIDRs, as these are reserved by Anypoint Platform:
```

Comment:
You cannot use the following list of CIDR blocks as private space network CIDRs:

Reply:
```suggestion
You cannot use the following list of CIDR blocks as private space network CIDRs, as these are reserved by Anypoint Platform:
```

Reply:
I don't think we need to make any change here regarding reserved CIDR feature, let's restore to its original content..

Reply:
Delete it or go with my original text?

Reply:
can we restore it to original text before the change for reserved cidrs?

Reply:
```suggestion
You cannot use the following list of CIDR blocks:
```